source/guides/maintenance/download-verify-decompress.rst

```diff
@@ -173,4 +173,4 @@ Image types
 .. include:: ../../reference/image-types.rst
    :start-after: incl-image-filename-end:
 
-.. _image: https://clearlinux.org/downloads
+.. _image: https://clearlinux.github.io/downloads
```

Comment:
I don't think this is supposed to be changed since it is going to the content part of the site. Anything under downloads probably needs to not change.